node-express-single-page-app/README.md

```diff
@@ -1,37 +1,32 @@
+This repo is an example application of the Intercom Messenger on a Node single page app using Express. You can follow along and look at how it is implemented in the code.
 
-## Check out the code above ⬆️
+On the **server side**, add an endpoint that will return identifying information about the currently logged in user to the Intercom Messenger.
 
-Check out the code in this repo to see an example of how to install the Messenger.
-
-## Summary
-
-1. Remove `window.intercomSettings` if you added it to your HTML in a previous installation step.
+    *You can also return the user information in the login response instead of a dedicated endpoint if you prefer.*
 
 1. Install the Intercom SDK (Server)
     ```
     npm i -s intercom-client
     ```
-1. Load `IdentityVerification` class (Server)
+1. Load the `IdentityVerification` class (Server)
     ```js
     var IdentityVerification = require('intercom-client').IdentityVerification;
     ```
-1. Load your user data (Server)
-1. Generate user hash using your user data with the Intercom SDK (Server)
-    ```js
-    var userHash = IdentityVerification.userHash({
-      secretKey: INTERCOM_SECRET_KEY,
-      identifier: req.user.id || req.user.email
-    });
-    ```
-1. Return the user data and hash from an endpoint and call `window.Intercom('boot', data)`, where data is the Intercom settings.
-    ```js
-    window.Intercom('boot', {
-      app_id: ...,
-      user_id: ...,
-      user_email: ...,
-      user_hash: ...
-    });
-    ```
+1. Add a route to your app called `/intercomSettings` ([see code](https://github.com/intercom/intercom-install-examples/blob/b61c0e4d1f3dd27a0bee3ee5688902355e48e348/node-express-single-page-app/app.js#L22)).
+1. Return an object to the client containing:
+   - The email of the currently logged user
+   - A hash based on the user’s email and your Intercom identity verification secret key like ([see code](https://github.com/intercom/intercom-install-examples/blob/b61c0e4d1f3dd27a0bee3ee5688902355e48e348/node-express-single-page-app/app.js#L23)). If you want more info on why this is important, check out [this doc](https://docs.intercom.com/configure-intercom-for-your-product-or-site/staying-secure/enable-identity-verification-on-your-web-product).
+
+**On the client side**, connect to Intercom and load the Messenger:
+1. Paste the code right before the closing body tag of every page where you want the Intercom Messenger to appear like ([see code](https://github.com/intercom/intercom-install-examples/blob/b61c0e4d1f3dd27a0bee3ee5688902355e48e348/node-express-single-page-app/public/index.html#L19).
+
+    *If you have already installed the Intercom Messenger on a page you want to track logged-in users on, remove the `window.intercomSettings` part as we will be using `window.Intercom(‘boot’, …)` to boot the Messenger.*
```

Comment:
It looks like the quotes around `boot` are smartquotes, which will break if used in code. We should change those to single quotes, as well as the double quotes around `update` in the next line.